crypto/txscript/test-data/script_tests-kip10.json

```diff
@@ -3746,18 +3722,6 @@
     "",
     "BAD_OPCODE"
   ],
-  [
-    "1",
-    "IF 0xc2 ELSE 1 ENDIF",
-    "",
-    "BAD_OPCODE"
-  ],
-  [
-    "1",
-    "IF 0xc3 ELSE 1 ENDIF",
-    "",
-    "BAD_OPCODE"
-  ],
```

Comment:
Can you keep test, and change result and description?

Reply:
commit 955424cd4c1dd72f4c1028b3bcdd3dd017bb6309